app/views/invitations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= flash[:message] %>
+
+<h1>Invite A User To An Event</h1>
+<p><<<<<<<<<<<<<<<>>>>>>>>>>>>>>></p>
+
+ <%= form_for :invitation, url:invitations_path do |f|%>
+
+  <%= f.label :invitee_id, 'Invitee ID'%>
+  <%= f.text_field :invitee_id, class:'form-control' %> <br/>
+
+  <%= f.label :attended_event_id, 'Event ID'%>
+  <%= f.text_field :attended_event_id, class:'form-control' %> <br/>
```

Comment:
How do I know the event id and user id of who I am inviting?

<img width="441" alt="Screenshot 2020-06-05 at 11 08 23" src="https://user-images.githubusercontent.com/31304943/83864697-0059d980-a71d-11ea-8a09-87397663b2a7.png">